lib/project.ts

```diff
@@ -149,7 +149,7 @@ export class Project implements Project.IProject {
 	}
 
 	private static IGNORE_FILE = ".abignore";
-	private static INTERNAL_NONPROJECT_FILES = [".ab", Project.IGNORE_FILE, "**/*.ipa", "**/*.apk", "**/*.xap"];
+	private static INTERNAL_NONPROJECT_FILES = [".ab", Project.IGNORE_FILE, ".*" + Project.IGNORE_FILE, "**/*.ipa", "**/*.apk", "**/*.xap"];
```

Comment:
What if the user has a file named "pencho.abproject", should we exclude it as well?